src/main/kotlin/ink/abb/pogo/scraper/util/map/Pokestop.kt

```diff
@@ -11,6 +11,7 @@ package ink.abb.pogo.scraper.util.map
 import com.pokegoapi.api.map.fort.Pokestop
 
 fun Pokestop.canLoot(ignoreDistance: Boolean = false, lootTimeouts: Map<String, Long>): Boolean {
-    val canLoot = lootTimeouts.getOrElse(id, { 0 }) < System.currentTimeMillis()
+    val now = System.currentTimeMillis()
+    val canLoot = lootTimeouts.getOrElse(id, { 0 }) < now && cooldownCompleteTimestampMs < now
```

Comment:
Wouldn't `val canLoot = lootTimeouts.getOrElse(id, { cooldownCompleteTimestampMs }) < now` work as well? As far as I know this `Map` is not filled with 0's or values it knows are incorrect.